queue-health/graph/graph.py

```diff
@@ -164,19 +168,21 @@ def render(history_lines, out_file):
         # Make them steps instead of slopes.
         if dts:
             dts.append(dt)
+
+            daily_happiness.append(daily_happiness[-1])
```

Comment:
Added a comment, it is what makes changes happen as steps (L) instead of slopes (\)